src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/GerritProjectInterestingTest.java

```diff
@@ -73,72 +74,110 @@ public static Collection getParameters() {
         topics.add(topic);
-        config = new GerritProject(CompareType.PLAIN, "project", branches, topics, null, null, false);
-        parameters.add(new InterestingScenario[]{new InterestingScenario(config, "project", "master", "topic", true)});
+        config = new GerritProject(CompareType.PLAIN, "project", branches, topics,
+                null, null, null, false);
+        parameters.add(new InterestingScenario[]{new InterestingScenario(config, "project", "master", "topic",
+                null, true), });
 
         topics = new LinkedList<Topic>();
         topic = new Topic(CompareType.ANT, "**/topic");
         topics.add(topic);
-        config = new GerritProject(CompareType.PLAIN, "project", branches, topics, null, null, false);
+        config = new GerritProject(CompareType.PLAIN, "project", branches, topics,
+                null, null, null, false);
         parameters.add(new InterestingScenario[]{new InterestingScenario(config,
-                "project", "master", "team/topic", true), });
+                "project", "master", "team/topic", null, true), });
 
         topics = new LinkedList<Topic>();
         topic = new Topic(CompareType.REG_EXP, ".*_topic");
         topics.add(topic);
-        config = new GerritProject(CompareType.PLAIN, "project", branches, topics, null, null, false);
+        config = new GerritProject(CompareType.PLAIN, "project", branches, topics,
+                null, null, null, false);
         parameters.add(new InterestingScenario[]{new InterestingScenario(config,
-                "project", "master", "team-wolf_topic", true), });
+                "project", "master", "team-wolf_topic", null, true), });
+
+        hashtags = new LinkedList<>();
+        Hashtag hashtag = new Hashtag(CompareType.PLAIN, "hashtag");
+        hashtags.add(hashtag);
+        config = new GerritProject(CompareType.PLAIN, "project", branches, null,
+                hashtags, null, null, false);
+        parameters.add(new InterestingScenario[]{new InterestingScenario(config,
+                "project", "master", null, List.of("hashtag"), true), });
+
+        hashtags = new LinkedList<>();
+        hashtag = new Hashtag(CompareType.ANT, "**/hashtag");
+        hashtags.add(hashtag);
+        config = new GerritProject(CompareType.PLAIN, "project", branches, null,
+                hashtags, null, null, false);
+        parameters.add(new InterestingScenario[]{new InterestingScenario(config,
+                "project", "master", null, List.of("test/hashtag"), true), });
+
+        hashtags = new LinkedList<>();
+        hashtag = new Hashtag(CompareType.REG_EXP, ".*_hashtag");
+        hashtags.add(hashtag);
+        config = new GerritProject(CompareType.PLAIN, "project", branches, null,
+                hashtags, null, null, false);
+        parameters.add(new InterestingScenario[]{new InterestingScenario(config,
+                "project", "master", null, List.of("test_hashtag"), true), });
```

Comment:
Missing test for when it should not trigger?

Reply:
add test for when should not trigger.